src/Authentication.jsx

```diff
@@ -0,0 +1,31 @@
+import AuthForm from "./AuthForm.jsx";
```

Comment:
Seems a little convoluted here.
You already track if a user is loggedin within the App.jsx

This is like a doubled up code base.
I understand whats its for but you could use React Router such that you dont even need this component.